strings/min_cost_string_conversion.py

```diff
@@ -67,17 +67,9 @@ def assemble_transformation(ops, i, j):
 			seq.append(ops[i][j])
 			return seq
 
-if __name__ == '__main__':
-	_, operations = compute_transform_tables('Python', 'Algorithms', -1, 1, 2, 2)
-
-	m = len(operations)
-	n = len(operations[0])
-	sequence = assemble_transformation(operations, m-1, n-1)
-
-	string = list('Python')
+def main(sequence,string):
```

Comment:
Don't add a main function which takes parameters (I didn't notice this earlier). Place all the code inside the `if __name__ == "__main__":` block into the main function block (`def main():`), but don't modify it other than that.

Reply:
Perhaps you could add a function like the one above, but with a different name, since it seems to be useful for your testing. Let's ask what the others think about that later.